L2/op_conductor_local_devnet.md

```diff
@@ -0,0 +1,149 @@
+export OP_CONDUCTOR_HEALTHCHECK_UNSAFE_INTERVAL=7200 # recommend a 2-3x multiple of your network block time to account for temporary performance issues
+export OP_CONDUCTOR_LOG_FORMAT=logfmt
+export OP_CONDUCTOR_LOG_LEVEL=debug
+export OP_CONDUCTOR_METRICS_ADDR=0.0.0.0
+export OP_CONDUCTOR_METRICS_ENABLED=true
+export OP_CONDUCTOR_METRICS_PORT=7300
+export OP_CONDUCTOR_ROLLUP_CONFIG=./rollup.json
+export OP_CONDUCTOR_NODE_RPC=http://localhost:8547 # your op-node RPC
+export OP_CONDUCTOR_RAFT_SERVER_ID=sequencer-0
+export OP_CONDUCTOR_RAFT_STORAGE_DIR=./conductor/raft
+export OP_CONDUCTOR_RPC_ADDR=0.0.0.0
+export OP_CONDUCTOR_RPC_ENABLE_ADMIN=true
+export OP_CONDUCTOR_RPC_ENABLE_PROXY=true
+export OP_CONDUCTOR_RPC_PORT=6660
+export OP_CONDUCTOR_PAUSED=false
+export OP_CONDUCTOR_RAFT_BOOTSTRAP=true
+
+# Start op-conductor
+./bin/op-conductor
+```
+5. Configure and Run [op-conductor-ops](https://github.com/ethereum-optimism/infra/tree/main/op-conductor-ops)
+```bash
+# Install dependencies. Follow readme.md to install poetry first
+poetry install
+
+# Configure op-conductor-ops
+cp example.config.toml config.toml
+
+# Example configuration
+[networks.op-network-N]
+sequencers = [
+    "sequencer-0",
+]
+[sequencers.sequencer-0]
+raft_addr = "<sequencer-0-ip-address>:50050"
+conductor_rpc_url = "http://<sequencer-0-ip-address>:6660"
+node_rpc_url = "http://<sequencer-0-ip-address>:8547"
+voting = true
+
+# Check the status of the first sequencer
+./op-conductor-ops status op-network-N
+```
+
+6. Add Additional Sequencers
+ - Deploy two sequencers (op-geth and op-node) on separate machines.
+     - Enable sequencer mode for op-node: OP_NODE_SEQUENCER_ENABLED=true
+     - Use the same sequencer key for op-node: OP_NODE_P2P_SEQUENCER_KEY
+     - Add op-node options:
+        ```bash
+        export OP_NODE_CONDUCTOR_ENABLED=true
+        export OP_NODE_CONDUCTOR_RPC=<conductor-rpc-endpoint>
+        ```
+     - To sync data from other sequencers, we need to add three options accroding to this [doc](https://github.com/ethstorage/pm/blob/main/L2/beta_testnet_new_node.md)
+        ```bash
+         --p2p.static # other two sequencers' address
+         --p2p.no-discovery
+         --p2p.sync.onlyreqtostatic
+         ```
+  - Wait for the new sequencer to start and get synced up with the rest of the nodes
+  - Once the new sequencer is synced up, manually or use automation to add it to the cluster by calling `conductor_addServerAsVoter` json rpc method on the leader sequencer
+      - ```bash
+        curl -X POST -H "Content-Type: application/json" --data \
+            '{"jsonrpc":"2.0","method":"conductor_addServerAsVoter","params":[<raft-server-id>, <raft-consensus-addr>, <raft-config-version>],"id":1}'  \
+            http://127.0.0.1:8547
+        ```
+  - Resume the conductor on the new sequencer by calling `conductor_resume` json rpc method on op-conductor
```

Comment:
What happens if `conductor_resume` is not called, The sequencer only syncs but never becomes the leader?

Reply:
Yes, the sequencer can only become the leader if both `conductor_resume` and `conductor_addServerAsVoter` are called